dist/litera/_bootswatch.scss

```diff
@@ -4,6 +4,7 @@
 
 // Variables ===================================================================
 
+// stylelint-disable-next-line value-keyword-case
 $font-family-serif:      Georgia, Cambria, "Times New Roman", Times, serif !default;
```

Comment:
@thomaspark is this supposed to be here and not in the theme's _variables.scss?

Reply:
Yup, it should be in _variables. Not sure how it got there...

Reply:
I'll make a separate PR for this.